src/lib/utils/queryUtil/index.ts

```diff
@@ -0,0 +1,83 @@
+interface PageQueryType {
+  latitude: number | null
+  longitude: number | null
+  zoom: number | null
+  places: number[] | null
+  showShadows: boolean | null
+  showTemperature: boolean | null
+  showWind: boolean | null
+  visibleHour: 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21 | 22 | 23 | null
```

Comment:
This should be from 10 to 21, shouldn't it?

Reply:
True! Let's merge that with the rest of the logic later! Ok?